src/Avalonia.Controls/ListBox.cs

```diff
@@ -29,18 +29,24 @@ public class ListBox : SelectingItemsControl
         /// <summary>
         /// Defines the <see cref="SelectedItems"/> property.
         /// </summary>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("AvaloniaProperty", "AVP1010",
+            Justification = "This property is owned by SelectingItemsControl, but protected there. ListBox changes its visibility.")]
```

Comment:
Again, this warning might be onto something. Maybe these should be made internal and made public in derived classes. Again, for another PR.